Automattic/pocket-casts-ios
# Transcripts: speaker info

| 📘 Part of: #1848  | 
|:---:|

This PR adds support for showing the speaker information on transcripts, if any is available.

| 1 | 2 |
| - | - |
| ![Simulator Screenshot - iPhone 15 - 2024-08-08 at 18 10 05](https://github.com/user-attachments/assets/fe4105a0-8589-4bf4-86cc-35ed9503e042) | ![Simulator Screenshot - iPhone 15 - 2024-08-08 at 16 06 56](https://github.com/user-attachments/assets/af11c1da-4ff6-41bd-8b33-93d888443398) |

## To test

- Start the app
- Ensure that you have the `transcripts` and `newShowNotesEndpoint` FF enabled
- Open this [episode](https://pca.st/8if3skl4) from Build Your SaaS that has transcript with speaker information
- Tap Play
- Open the full screen player
- Tap on transcripts
- Check that the transcript information is show correctly and you see the speakers info correctly:
  - Speaker names shows in its own line with a smaller font
  - Speaker names are not repeated in sequential paragraphs, new speaker information should only be show when a new speaker is talking
  - Speaker name have an empty line before then.
  -  Speaker name font size should change when accessibility font size is changed on Settings.

## Checklist
